### ISSUE TITLE
Event replay search does not execute the correct search query when the event is doing a group by on a field that does not exist

### ISSUE THREAD

**Issue by drewmiranda-gl** [Graylog2/graylog2-server#17496]
Event replay search does not execute the correct search query when the event is doing a group by on a field that does not exist. While it won't be normal to intentionally create an event definition that does a group by aggregation on a field we told our search query to exclude, i put this together as a test to reproduce what I found.

<img width="301" alt="image" src="https://github.com/Graylog2/graylog2-server/assets/107503402/7bd4722d-715e-4771-abd7-ac6acf84ada5">

<br>

<img width="185" alt="image" src="https://github.com/Graylog2/graylog2-server/assets/107503402/1df8a63e-504a-44f9-af3e-818dd563e19f">

<br>

<img width="1378" alt="image" src="https://github.com/Graylog2/graylog2-server/assets/107503402/0ba8f652-6400-490f-add9-98d4faaafa42">

## Expected Behavior

When a field is not present, the query should be `AND (-_exists_:user_name)`

## Current Behavior

When a field is not present, the query is `AND (user_name:"(Empty Value)")`

## Steps to Reproduce (for bugs)

1. Create a new Graylog Event
    * Search Query: `_exists_:message AND -_exists_:user_name`
    * Aggregation / Group by Field(s): `user_name`
2. Save event definition and wait for it to be triggered
3. On the "Alerts & Events" page click "Replay search"

## Context

I was working with a prospect and testing an event definition with them. We noticed that some of the triggered events were for messages that did not have the field we used in our group by aggregation and when we clicked on replay search, zero messages were shown on the search page.

## Your Environment

* Graylog Version: 5.2.1
* Java Version: Bundled
* OpenSearch Version: 2.11
* MongoDB Version: 6.0.11
* Operating System: Ubuntu Server 22.04 LTS
* Browser version: Google Chrome Version 119.0.6045.159 (Official Build) (arm64)

Please let me know if there are any questions. Thanks!

**Comment by kroepke** [comment]
Possibly related to #17367 and #17551

**Comment by patrickmann** [comment]
I don't think this is related to the `card() `issues https://github.com/Graylog2/graylog2-server/issues/17367 and https://github.com/Graylog2/graylog2-server/issues/17551

It is related to #17314. They should probably be addressed at the same time.

**Comment by waab76** [comment]
@kmerz @Emy-01 This is related to an open support issue: https://github.com/Graylog2/support/issues/104

Any estimate on when it might be looked at?

**Comment by janheise** [comment]
@drewmiranda-gl can I ask you to re-test this with a current version? The attached screenshot is from my local dev system with no additional changes since I fixed a different bug on Event replay a couple weeks ago. Maybe this one is by now fixed, too?

<img width="1370" alt="Image" src="https://github.com/user-attachments/assets/bb1694a0-e228-4ea8-a762-1b57c3462377" />

**Comment by drewmiranda-gl** [comment]
Hi @janheise , I just tested and can conform i no longer encounter this bug in Graylog 6.2.2. Thanks!
